ext/pdo/pdo_dbh.c

```diff
@@ -1423,6 +1424,13 @@ void pdo_dbh_init(void)
 
 	REGISTER_PDO_CLASS_CONST_LONG("CURSOR_FWDONLY", (zend_long)PDO_CURSOR_FWDONLY);
 	REGISTER_PDO_CLASS_CONST_LONG("CURSOR_SCROLL", (zend_long)PDO_CURSOR_SCROLL);
+
+	zend_add_parameter_attribute(
+		zend_hash_str_find_ptr(&pdo_dbh_ce->function_table, "__construct", sizeof("__construct") - 1),
+		2,
+		zend_ce_sensitive_parameter->name,
+		0
+	);
```

Comment:
It would be nice if this could be done through ArgInfo and the stubs at some point.

Reply:
I agree, this would be nice. For now I've dropped adding the attribute to existing functions from this PR. This nicely separates the implementation of the behavior from adding the attribute to functions, making this easier to review.